typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -35,7 +35,7 @@
 
 OLD_GENERICS = False
 try:
-    from typing import _type_vars, _next_in_mro, _type_check  # noqa
+    from typing import _type_vars, _next_in_mro, _type_check # noqa
```

Comment:
```suggestion
    from typing import _type_vars, _next_in_mro, _type_check  # noqa
```

Reply:
> ... Or maybe just call that

Yes, that makes the change significantly smaller - thanks